### PR TITLE
Make plotObservationPeriod labels more clear

### PR DESCRIPTION
**Pull request by martaalcalde**

### CODE REVIEW

**Comment by martaalcalde** [comment]
Fixes #180

File: R/checks.R [hunk 1]
```diff
@@ -154,8 +154,19 @@ validateStudyPeriod <- function(cdm, studyPeriod, call = parent.frame()) {
...
+validateFacet <- function(facet, result, call = parent.frame()) {
+
+  if(rlang::is_formula(facet)){
+    facet <- as.character(facet)
+    facet <- unlist(strsplit(facet, " \\+ "))
+    facet <- facet[facet != "~" & facet != "+" & facet != "."]
+  }
+    # Assert choices
+    optFacetColour <- visOmopResults::tidyColumns(result)
+    optFacetColour <- optFacetColour[optFacetColour %in% visOmopResults::tidyColumns(result)]
```
Comment (catalamarti) [inline R/checks.R:166]: ?

File: R/plotObservationPeriod.R [hunk 2]
```diff
@@ -52,31 +55,37 @@ plotObservationPeriod <- function(result,
...
-  omopgenerics::assertChoice(asCharacterFacet(facet), optFacetColour)
-  colour <- optFacetColour[!optFacetColour %in% facet]
+  omopgenerics::assertChoice(facet, optFacetColour, null = TRUE, call = call)
 
   # this is due to bug in visOmopResults to remove in next release
   # https://github.com/darwin-eu/visOmopResults/issues/246
   if (length(facet) == 0) facet <- NULL
   if (length(colour) == 0) colour <- NULL
 
+  if(length(unique(result$group_name)) == 1){
```
Comment (catalamarti) [inline R/plotObservationPeriod.R:69]: ?
maybe use length(visOmopResults::groupColumns()) == 0?

**Review by catalamarti — COMMENTED** [review]
hey @martaalcalde can you comment on the two points I do not understand?
I don't know if I understood what you were trying to do...

File: R/checks.R (hunk 1 above)
Comment (martaalcalde) [reply R/checks.R:166]: These lines check if the facet argument includes columns from the tidyColumns

Comment (martaalcalde) [reply R/checks.R:166]: I've simplified it in the next pr

**Pull request merged**